client/components/BarVersion/BarVersion.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../stylesheets/colors';
+@import '../../../stylesheets/variables';
+
+.bar-graph__bar-version {
```

Comment:
This solution produces a weird alignment for a package like `react-pdf@4.2.0-beta`

<img width="660" alt="Screenshot 2022-11-03 at 8 32 04 PM" src="https://user-images.githubusercontent.com/8946207/199757595-8ddfbb7a-4a0b-4f14-bb1b-d85411133865.png">

Ideally all of these versions would stick up to the top? Arguable.